docs/modules/operation/pages/provisioning/detectors/HostResourceSWRunDetector.adoc

```diff
@@ -0,0 +1,40 @@
+=== HostResourceSWRunDetector
+
+This detector test the running state of one or more processes.
+It does this via SNMP by inspecting the _hrSwRunTable_ of the http://www.ietf.org/rfc/rfc2790[HOST-RESOURCES-MIB].
```

Comment:
```suggestion
Use the http://www.ietf.org/rfc/rfc2790[HOST-RESOURCES-MIB] to detect a process to monitor.
```
Is the process doing the monitoring, or are we detecting a process to monitor?

Reply:
We detect a running process to be able to monitor it with [HostResourceSwRunMonitor](https://github.com/OpenNMS/opennms/blob/develop/docs/modules/operation/pages/service-assurance/monitors/HostResourceSwRunMonitor.adoc).
The process is listed in the HOST-RESOURCES-MIB.

Reply:
Thanks. Then my suggestion is phrased correctly.